### ISSUE TITLE
caffe 1.0 version ,  CPU only and no opencv , run  examples/mnist  why accuracy = 0.1076  ?help!!!

### ISSUE THREAD

**Issue by bugwang** [BVLC/caffe#7089]
I20250403 16:40:52.338333 720070 solver.cpp:329] Iteration 9000, Testing net (#0)
I20250403 16:40:55.531296 720070 solver.cpp:396]     Test net output #0: accuracy = 0.1109
I20250403 16:40:55.531350 720070 solver.cpp:396]     Test net output #1: loss = 8.02739 (* 1 = 8.02739 loss)
I20250403 16:40:55.650857 720070 solver.cpp:217] Iteration 9000 (11.7357 iter/s, 8.521s/100 iters), loss = 0.209952
I20250403 16:40:55.650914 720070 solver.cpp:236]     Train net output #0: loss = 0.209952 (* 1 = 0.209952 loss)
I20250403 16:41:02.860724 720070 solver.cpp:217] Iteration 9100 (13.8715 iter/s, 7.209s/100 iters), loss = 0.0648483
I20250403 16:41:02.860821 720070 solver.cpp:236]     Train net output #0: loss = 0.0648483 (* 1 = 0.0648483 loss)
I20250403 16:41:08.115089 720070 solver.cpp:217] Iteration 9200 (19.0331 iter/s, 5.254s/100 iters), loss = 0.0660819
I20250403 16:41:08.115135 720070 solver.cpp:236]     Train net output #0: loss = 0.0660818 (* 1 = 0.0660818 loss)
I20250403 16:41:13.373898 720070 solver.cpp:217] Iteration 9300 (19.0186 iter/s, 5.258s/100 iters), loss = 0.14472
I20250403 16:41:13.373957 720070 solver.cpp:236]     Train net output #0: loss = 0.144719 (* 1 = 0.144719 loss)
I20250403 16:41:18.628131 720070 solver.cpp:217] Iteration 9400 (19.0331 iter/s, 5.254s/100 iters), loss = 0.145094
I20250403 16:41:18.628175 720070 solver.cpp:236]     Train net output #0: loss = 0.145094 (* 1 = 0.145094 loss)
I20250403 16:41:23.830125 720070 solver.cpp:329] Iteration 9500, Testing net (#0)
I20250403 16:41:26.986752 720070 solver.cpp:396]     Test net output #0: accuracy = 0.1363
I20250403 16:41:26.986799 720070 solver.cpp:396]     Test net output #1: loss = 8.1345 (* 1 = 8.1345 loss)
I20250403 16:41:27.095007 720070 solver.cpp:217] Iteration 9500 (11.812 iter/s, 8.466s/100 iters), loss = 0.26221
I20250403 16:41:27.095068 720070 solver.cpp:236]     Train net output #0: loss = 0.26221 (* 1 = 0.26221 loss)
I20250403 16:41:42.050647 720070 solver.cpp:217] Iteration 9600 (6.68673 iter/s, 14.955s/100 iters), loss = 0.147457
I20250403 16:41:42.050733 720070 solver.cpp:236]     Train net output #0: loss = 0.147457 (* 1 = 0.147457 loss)
I20250403 16:41:47.319715 720070 solver.cpp:217] Iteration 9700 (18.9825 iter/s, 5.268s/100 iters), loss = 0.134561
I20250403 16:41:47.319772 720070 solver.cpp:236]     Train net output #0: loss = 0.13456 (* 1 = 0.13456 loss)
I20250403 16:41:52.592808 720070 solver.cpp:217] Iteration 9800 (18.9645 iter/s, 5.273s/100 iters), loss = 0.0875521
I20250403 16:41:52.592870 720070 solver.cpp:236]     Train net output #0: loss = 0.087552 (* 1 = 0.087552 loss)
I20250403 16:41:57.857460 720070 solver.cpp:217] Iteration 9900 (18.997 iter/s, 5.264s/100 iters), loss = 0.0790813
I20250403 16:41:57.857512 720070 solver.cpp:236]     Train net output #0: loss = 0.0790812 (* 1 = 0.0790812 loss)
I20250403 16:42:03.075994 720070 solver.cpp:446] Snapshotting to binary proto file examples/mnist/lenet_iter_10000.caffemodel
I20250403 16:42:03.083982 720070 sgd_solver.cpp:271] Snapshotting solver state to binary proto file examples/mnist/lenet_iter_10000.solverstate
I20250403 16:42:03.111616 720070 solver.cpp:309] Iteration 10000, loss = 0.212135
I20250403 16:42:03.111658 720070 solver.cpp:329] Iteration 10000, Testing net (#0)
I20250403 16:42:05.909335 720070 solver.cpp:396]     Test net output #0: accuracy = 0.1076
I20250403 16:42:05.909391 720070 solver.cpp:396]     Test net output #1: loss = 8.08709 (* 1 = 8.08709 loss)
I20250403 16:42:05.909404 720070 solver.cpp:314] Optimization Done.
I20250403 16:42:05.909411 720070 caffe.cpp:267] Optimization Done.

**Comment by dragonQian** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
